src/reducers/authReducer.jsx

```diff
@@ -39,7 +39,7 @@ const authReducer = (state = initState, action) => {
     case "FACEBOOK_AUTH_ERROR":
       return {
         ...state,
-        authError: "Facebook authentication failed!",
+        authError: "Facebook authentication failed..!",
```

Comment:
please delete the two dots after the messages, usually this is not present in messages shown to the user. and consider deleting the exclamation points (most sites do not do this either).